odk2-src/build-app.rst

```diff
@@ -0,0 +1,778 @@
+      $('#FIELD_1').attr('checked', true);
+    }
+    $('#FIELD_1').attr('disabled', true);
+
+  }
+
+  /**
+   * Assumes censusResultSet has valid content
+   *
+   * Updates document field with the value for the elementKey
+   */
+  function nullCaseHelper(elementKey, documentSelector) {
+    var temp = censusResultSet.get(elementKey);
+    if (temp !== null && temp !== undefined) {
+      $(documentSelector).text(temp);
+    }
+  }
+
+As with the *List View*, this view requires a separate CSS file. Copy the following file from a default Application Designer, maintaining the directory path in this application's directory:
+
+  - :file:`config/assets/css/detail.css`
+
+
+.. _build-app-creating-web-file-properties:
+
+Defining Default View Files
+""""""""""""""""""""""""""""
+
+The :file:`.xlsx` form should be updated to indicate the default view type, and where to find the HTML files for *Detail View* and *List View*. Open :file:`app/config/tables/census/forms/census/census.xlsx` and add a new worksheet titled *properties*. Add the following headers: :th:`partition`, :th:`aspect`, :th:`key`, :th:`type`, and :th:`value`.
+
+Add the following rows to set your *List View* and *Detail View* default files:
+
+.. list-table:: *properties* worksheet
+  :header-rows: 1
+
+  * - partition
+    - aspect
+    - key
+    - type
+    - value
+  * - Table
+    - default
+    - defaultViewType
+    - string
+    - LIST
+  * - Table
+    - default
+    - detailViewFileName
+    - string
+    - config/tables/census/html/census_detail.html
+  * - Table
+    - default
+    - listViewFileName
+    - string
+    - config/tables/census/html/census_list.html
+
+See :ref:`xlsx-ref-properties` for more details about specifying custom HTML files.
+
+Run :file:`census.xlsx` through the XLSX Converter again (:ref:`build-app-generate-formdef`) to update the configuration.
+
+After that, you can deploy your app to your device. Open Survey and fill in a few census records. Then, open Tables and select the *Census* table. This should automatically launch the *List View* defined above. Tapping an item in the *List View* should launch the detail view.
+
+.. _build-app-debugging-tables:
+
+Debugging Tables Web Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use the :program:`Chrome` browser on your computer to inspect for devices and connect to this custom screen on your Android device, and debug from there.
+
+.. warning::
+  The edit-debug cycle is awkward because you must make the HTML or JavaScript change on your computer then push the change to your device, and reload the page (for example, by rotating the screen). When you do rotate the screen, however, it is rendered in a new web page, necessitating connecting to that new page to resume debugging (the prior page sits idle and will eventually be destroyed. If you don't see any activity, it is likely because you are pointing at the wrong web page. Return to inspect devices, and select the newest page).
```

Comment:
Would a reference to a Google resource help here?  Something like https://developers.google.com/web/tools/chrome-devtools/javascript?

Reply:
Or the Android Remote Debugging page? 
https://developers.google.com/web/tools/chrome-devtools/remote-debugging/